Source/Directory.Build.props

```diff
@@ -1,6 +1,6 @@
 <Project>
  <PropertyGroup>
-   <LangVersion>latest</LangVersion>
+   <LangVersion>preview</LangVersion>
```

Comment:
Note to @rockfordlhotka - add a work item to reset this to `latest` once .NET 9 is released.

Reply:
@rockfordlhotka fixed in https://github.com/MarimerLLC/csla/pull/4302